partner_property/migrations/16.0.1.1.0/pre-migration.py

```diff
@@ -15,7 +15,23 @@ def migrate(env, version):
         WHERE key = 'partner_property.properties_definition'
         """,
     )
-    icp = env["ir.config_parameter"].get_param(
-        "partner_property.properties_definition_company"
+    openupgrade.logged_query(
+        env.cr,
+        """
+        INSERT INTO ir_config_parameter (
+            create_uid,
+            write_uid,
+            key,
+            value,
+            create_date,
+            write_date
+        ) SELECT create_uid,
+            write_uid,
+            'partner_property.properties_definition_person',
+            value,
+            create_date,
+            write_date
+            FROM ir_config_parameter
+            WHERE key = 'partner_property.properties_definition_company'
+        """,
     )
-    icp.copy({"key": "partner_property.properties_definition_person"})
```

Comment:
Why not doing the copy before the renaming by SQL?

Reply:
Because the `get_param()` method returns the value, not the record.

Reply:
You can do a search as well.